aiohttp/helpers.py

```diff
@@ -484,13 +484,13 @@ def _format_r(request: 'BaseRequest',
     @staticmethod
     def _format_s(request: 'BaseRequest',
                   response: 'StreamResponse',
-                  time: float) -> str:
+                  time: float) -> int:
```

Comment:
Correct if I am wrong, but doing typings in `web_response.py` reveled that it should be `int` most likely.

Reply:
You are right.
As an option, we can return `str` but should apply `str(response.status)`.
Let's change annotation as you did already.